scripts/init.d-xapissl

```diff
@@ -27,6 +27,15 @@ XAPISSL_LOCK="/var/lock/xapissl"
 BACK_COMPAT=0
 [ "${3}" = back_compat_6_5 ] && BACK_COMPAT=1
 
+# Check if there is request to overwrite permfile
+_PARAM="${4}"
+if [ -n "${_PARAM}" ]; then
+    _PERMFILE="${_PARAM#*=}"
+    if [ -n "${_PERMFILE}" ] && [ -e "${_PERMFILE}" ]; then
```

Comment:
This can be combined into
```
if [ -n "${_PERMFILE}" -a -e "${_PERMFILE}" ]; then
```

Why is the variable called PERMFILE (with an R) to hold a value for PEMFILE?